### PR TITLE
Cleanup lambda

### PR DESCRIPTION
**Pull request by jackmurdoch**

### CODE REVIEW

File: cleanup.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  const { rows } = await client.query(sql)
+  const [{ value }] = rows
+
+  console.log(`updated register metric for today with value ${value}`)
+}
+
+async function removeExpiredCodes(client, codeLifetime) {
+  const sql = SQL`
+    DELETE FROM verifications
+    WHERE created_at < CURRENT_TIMESTAMP - ${`${codeLifetime} mins`}::INTERVAL
```
Comment (colmharte) [inline cleanup.js:23]: This needs to be a longer interval as we want users to get the expired code messages.

**Review by colmharte — CHANGES_REQUESTED** [review]

File: cleanup.js (hunk 1 above)
Comment (jackmurdoch) [reply cleanup.js:23]: That's fair enough - shall we hard code or create another param in infra to manage this? Or we could just keep the code around for `codeLifetime * 3` or something similar.

Comment (colmharte) [reply cleanup.js:23]: I think a separate param to configure how long we keep for.

**Comment by jackmurdoch** [comment]
Will merge after https://github.com/nearform/covid-green-infra/pull/30 has been merged and applied.

**Pull request merged**